CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## main
 
+### Packaging
+* MapboxNavigation now requires [MapboxMaps v10.8._x_](https://github.com/mapbox/mapbox-maps-ios/releases/tag/v10.8.0-beta.1). ([#4085](https://github.com/mapbox/mapbox-navigation-ios/pull/4085))
+* MapboxCoreNavigation now requires [MapboxNavigationNative v112._x_](https://github.com/mapbox/mapbox-navigation-native-ios/releases/tag/112.0.0). ([#4085](https://github.com/mapbox/mapbox-navigation-ios/pull/4085))
```

Comment:
This release comes with new `EdgeMetadata.isUrban` and `RoadObject.isUrban` properties that we should hook up in `Metadata.init(_:)` and `RoadObject.init(_:)` and expose via new `Metadata.isUrban` and `RoadObject.isUrban` properties, or we could leave them as tail work by tracking them in a separate ticket.

Reply:
I'll include these new properties.

Reply:
Apparently `MapboxNavigationNative.RoadObject.isUrban` is optional because it only makes sense for certain kinds of road objects. Ideally, we would expose it as an associated value for specific cases of `RoadObject.Kind` or as an associated value of `RoadObject.Location.point(_:)`, not as a property on `RoadObject` itself. However, it looks like it can also be `nil` if the road object comes from a route response, representing a route alert. In principle, any kind of location could be considered urban or non-urban, so leaving it as a property on `RoadObject` would be acceptable, if less elegant and more error-prone.